manual_tests/ch32838_test/input.json

```diff
@@ -0,0 +1,34 @@
+{
+    "build_volume": [50, 50, 50],
+    "spacing": 1,
+    "items": [
+        {
+            "filename": "./manual_tests/ch32838_test/vertical_solid_arrow.stl",
+            "count": 14,
+            "scale": 1.0,
+            "copack": [
+                {
+                    "filename": "./manual_tests/ch32838_test/arrow-minted_cuboid.stl"
+                },
+                {
+                    "filename": "./manual_tests/ch32838_test/vertical_solid_arrow__translated.stl"
+                }
+            ]
+        },
+        {
+            "filename": "./manual_tests/ch32838_test/logo.stl",
+            "count": 21,
+            "scale": 1.0
+        },
+        {
+            "filename": "./manual_tests/ch32838_test/arrow-minted_cuboid.stl",
+            "count": 10,
+            "scale": 1.0,
+            "copack": [
+                {
+                    "filename": "./manual_tests/ch32838_test/vertical_solid_arrow.stl"
+                }
+            ]
+        }
+    ]
+}
```

Comment:
`manual_tests/ch32838_logo.stl` or `manual_tests/ch32838_test_logo.stl`  I think renaming tests 👍 . All test file should be in one directory, for simplicity. names from shortcut / clubhouse tickets are a solid idea.

Reply:
Okay, there's also an additional folder `tests` that contains super-basic tests that run in Jenkins at each push.